### DISCUSSION TITLE
Maximum line lengt - PEP8 = 79 characters; 72 characters for docstring comments

### DISCUSSION THREAD

**Discussion by Hermann12** [thonny/thonny#3236] [General]
Thonny V4.1.4 portable; have a drop box under _Tools | Options | Editor_ to set a grey border info for code line length.
But the recommended size couldn't be set exactly as [PEP8](https://peps.python.org/pep-0008/) recommend, because the steps are ... 0, 60, 70, 80, 88, 90, 100 ....
It would be nice, if the user can set a free value e.g. 72, 79 and maybe you have two (light grey, dark gray) lines?
Is there a possibility to adjust the standard steps/values, maybe in a .ini file etc.? Thanks a lot!

**Comment by aivarannamaa** [comment]
Thanks for the suggestion and sorry for the late reply.

Free values are more hassle for me, as I need to perform validation to the value. I simply added 72 and 79 as choices now. It will be published in version 5.0

**Reply by JohnJSal** [reply]
I had this same question/request and found this discussion from a couple of years ago. I have version 5.0, but I don't see the options for 72 and 79.

Has this update been made yet?

Thanks!

**Reply by aivarannamaa** [reply]
Most what was meant to be in version 5.0, was postponed to 6.0, including those mentioned here.

**Reply by JohnJSal** [reply]
Ok, thanks for the response!
